sc/src/main/java/uk/ac/ebi/atlas/home/JsonLatestExperimentController.java

```diff
@@ -0,0 +1,22 @@
+package uk.ac.ebi.atlas.home;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.ac.ebi.atlas.controllers.JsonExceptionHandlingController;
+import uk.ac.ebi.atlas.trader.ScxaExperimentTrader;
+import javax.inject.Inject;
+
+@RestController
+public class JsonLatestExperimentController extends JsonExceptionHandlingController {
+    private final LatestExperimentsService latestExperimentsService;
+    @Inject
+    public JsonLatestExperimentController (LatestExperimentsDao latestExperimentsDao, ScxaExperimentTrader experimentTrader) {
```

Comment:
You should inject here the `latestExperimentsService`, annotate the service to be a `@Component` and have the DAO and the trader injected there, so [the IoC chain](https://docs.spring.io/spring/docs/4.3.x/spring-framework-reference/html/overview.html#overview-dependency-injection) follows naturally.

Reply:
Remember to fix the constructor.